### ISSUE TITLE
Wrong version of `multiformats` package breaking `@lit-protocol/contracts-sdk`

### ISSUE THREAD

**Issue by gilbert** [LIT-Protocol/js-sdk#403]
### Describe the bug
Lit's build system is breaking bundling with vite/rollup. Specifically:

- [contract-sdk's multiformats version](https://github.com/LIT-Protocol/js-sdk/blob/6d1e1188bf9923099b998c6f146dfd9f58de7239/packages/contracts-sdk/package.json#L29) is set to `^9.7.1`, but **only** in peerDependencies.
- Other packages, such as core, have `multiformats` version set to `^12.1.3`, likely coming from the [root package.json](https://github.com/LIT-Protocol/js-sdk/blob/6d1e1188bf9923099b998c6f146dfd9f58de7239/package.json#L127) (not familiar with how Nx works)
- This causes, during vite bundling, the contracts-sdk code to attempt a `require` on multiformats v12.x, **which is problematically an esm module**, breaking the build with this error:

```
No known conditions for "./cid" specifier in "multiformats" package [plugin vite:dep-pre-bundle]

node_modules/@lit-protocol/contracts-sdk/src/lib/contracts-sdk.js:33:18:
      33 │     CID = require('multiformats/cid');
```

### To Reproduce
See [this minimal example repo](https://github.com/gilbert/lit-vite-bug-minimal).

### Suggested Fix

Either update contracts-sdk to dynamic import the existing multiformats esm version (I can provide a pr), or move the older multiformats dependency from `peerDependencies` to `dependencies`.

**Comment by wakechromium** [comment]
Getting same error with nextjs also

**Comment by Ansonhkg** [comment]
Thanks for reporting this @gilbert! 

I've created a PR here: https://github.com/LIT-Protocol/js-sdk/pull/404/files

**Comment by Ansonhkg** [comment]
The changes are now published to 4.1.0 194bcf95a027a289e8e96e506a32d74c31bd38db. Please let me know if they work for you! :)

**Comment by MaximusHaximus** [comment]
Closing this as the fix has been shipped -- if you're still having any issues, just let us know and we can re-open :)
